src/language-js/printer-estree.js

```diff
@@ -2722,13 +2722,20 @@ function printPathNoParens(path, options, print, args) {
             concat([
               options.bracketSpacing ? line : softline,
               n.readonlyToken
-                ? concat([path.call(print, "readonlyToken"), " "])
+                ? concat([
+                    n.readonlyToken.type === "TSMinusToken"
```

Comment:
We should support `TSPlusToken` simultaneously.

Reply:
For reference, the corresponding TypeScript PR --> Microsoft/TypeScript#21919

Reply:
Oh, sorry, I don't know it is a thing. Will also add TSPlusToken

Reply:
@ikatyang Added.